os/board/rtl8730e/src/rtl8730e_ndp120.c

```diff
@@ -131,6 +131,16 @@ static void rtl8730e_ndp120_set_dmic(bool enable)
 	}
 }
 
+static void rtl8730e_ndp120_pm(bool sleep)
+{
+	/* h/w specific things to be done if required */
+	if (sleep) {
+		lldbg("board entering sleep\n");
```

Comment:
Is there any reason why you use the lldbg instead of xxdbg?

Reply:
I have added it just for testing purpose. I have updated it now to use vdbg isntead.